tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpNoHearbeatsTest.java

```diff
@@ -137,18 +139,21 @@ public void inspectOpenedResource(Connection connection) {
    }
 
    public static void main(String[] arg) {
-      if (arg.length > 0 && arg[0].equals("testConnectionReset")) {
+      if (arg.length > 0 && arg[0].startsWith("testCloseConsumerOnConnectionReset")) {
```

Comment:
How about using the argument as the queue name? That way you dont need a constant, and there are side benefits such as test isolation and if things to wonky you can read the logs and immediately know what test the output came from etc?

Reply:
I was going to do that, but then the argument was: testCloseConsumerOnConnectionReset[useOverride=true]() 

The test is using parameters.. which I didn't like as the name of the queue... 

I would prefer leaving it as is...